sql.py

```diff
@@ -1636,24 +1636,28 @@ def insertTxAddr(rawtx, Protocol, TxDBSerialNum, Block):
           else:
             PropertyID=0
 
-        Ecosystem=getEcosystem(PropertyID) 
+        Ecosystem=getEcosystem(PropertyID)
 
         if txtype in [53,70,-1,25,26,27,28,185,186,65534,65535]:
           value=0
           value_neg=0
-        else: 
+        else:
           #if rawtx['result']['divisible']:
-          if getDivisible(rawtx):
-            value=int(decimal.Decimal(str(rawtx['result']['amount']))*decimal.Decimal(1e8))
-          else:
-            value=int(rawtx['result']['amount'])
+          try:
```

Comment:
In the testnet DB there is a transaction with an "amount" that throws this error: `Invalid literal for Decimal: '-92233720368.-54775808'`

This just makes it a bit more robust.